exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -835,13 +1007,76 @@ func (ku *Kucoin) CancelAllOrders(ctx context.Context, orderCancellation *order.
 	var values []string
 	switch orderCancellation.AssetType {
 	case asset.Margin, asset.Spot:
-		tradeType := ku.accountToTradeTypeString(orderCancellation.AssetType, marginModeToString(orderCancellation.MarginType))
-		values, err = ku.CancelAllOpenOrders(ctx, pairString, tradeType)
+		orderIDs := []string{}
+		if orderCancellation.OrderID != "" {
+			orderIDs = append(orderIDs, orderCancellation.OrderID)
+		}
+		if orderCancellation.ClientOrderID != "" {
+			orderIDs = append(orderIDs, orderCancellation.ClientOrderID)
+		}
+		switch orderCancellation.Type {
+		case order.OCO:
```

Comment:
I would like to see test cases for all of these new order types that result in API calls